### PR TITLE
block winner in archive db

### LINKED ISSUE #6834: Make "delegator win" appear in blocks in archive node (put to non-develop branch)

### PR DESCRIPTION
**Pull request by deepthiskumar**
Added block winner field to the blocks table in archive db. This field represents the winner (delegator/staker) of the block.

for a hardfork the archive db needs be updated with the following script 
```sql
--add the column without the not null constraint first
alter table blocks add column block_winner_id int;

--update the values of the new column. Making winner = creator because there is no way to get this information
update blocks as b1 set block_winner_id = (select creator_id from blocks b2 where b1.id = b2.id);

--add the not null, foreign key constraint
alter table blocks alter column block_winner_id set not null;
alter table blocks add constraint block_winner_id_fkey foreign key (block_winner_id) references public_keys(id)
```

Checklist:

- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them:

Closes #6834

### CODE REVIEW

**Review by mrmr1993 — APPROVED** [review]

**Review by psteckler — APPROVED** [review]

**Review by lk86 — APPROVED** [review]
Lgtm if the rosetta + graphql changes pass a product review

File: src/lib/coda_graphql/coda_graphql.ml [hunk 1]
```diff
@@ -1189,6 +1189,17 @@ module Types = struct
             ~args:Arg.[]
             ~resolve:(fun {ctx= coda; _} {With_hash.data; _} ->
               AccountObj.get_best_ledger_account_pk coda data.creator )
+        ; field "winner" ~typ:(non_null public_key)
+            ~doc:"Public key of account that won the slot (Delegator/staker)"
+            ~deprecated:(Deprecated (Some "use creatorAccount field instead"))
```
Comment (bkase) [inline src/lib/coda_graphql/coda_graphql.ml:1194]: Can you change this to winnerAccount instead? (And make sure you don’t have deprecated)

**Review by bkase — CHANGES_REQUESTED** [review]

File: src/lib/coda_graphql/coda_graphql.ml (hunk 1 above)
Comment (deepthiskumar) [reply src/lib/coda_graphql/coda_graphql.ml:1194]: There is a winnerAccount below, so just removing this one

**Review by bkase — APPROVED** [review]
Yes can you add the winner to the set? Thanks!

**Pull request merged**